moabb/pipelines/utils.py

```diff
@@ -82,15 +82,26 @@ def parse_pipelines_from_directory(dir_path):
             # load config
             config_dict = yaml.load(content, Loader=yaml.FullLoader)
             ppl = create_pipeline_from_config(config_dict["pipeline"])
-            pipeline_configs.append(
-                {
-                    "paradigms": config_dict["paradigms"],
-                    "pipeline": ppl,
-                    "name": config_dict["name"],
-                }
-            )
+            if config_dict["param_grid"] is not None:
```

Comment:
```suggestion
            if "param_grid" in config_dict and config_dict["param_grid"] is not None:
```
With this check, you could process yaml files without param_grid entry. Could you then remove the param_grid from the pipelines where it is not necessary?